### PR TITLE
GAWB-2801: Use workbench-libs for API calls in automation (Thurloe only) [definitely don't merge]

### PR DESCRIPTION
**Pull request by MatthewBemis**
Right now this doesn't compile because we are mixing different AuthToken types. Once Sam, Orch, and Rawls are completed we can move over to the workbench AuthToken fully.

- [ ] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: If you changed a URL that is used elsewhere (e.g. in an email), comment about where it is used and ensure the dependent code is updated.
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [ ] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [ ] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [ ] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [ ] **Submitter**: If you're adding new automated UI tests, review the test plan with QA
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Product Owner** sign off
- [ ] **Submitter**: Verify all tests go green, including CI tests and automated UI tests.
- [ ] **Submitter**: Squash commits and merge to develop. If adding test code, merge application code and test code at the same time.
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: automation/project/Dependencies.scala [hunk 1]
```diff
@@ -12,6 +12,9 @@ object Dependencies {
   val workbenchGoogleV = "0.10-d97f551"
   val workbenchGoogle: ModuleID = "org.broadinstitute.dsde.workbench" %% "workbench-google" % workbenchGoogleV excludeAll excludeWorkbenchModel
 
+  val workbenchServiceV = "0.0.1-alpha-b350837-SNAP"
```
Comment (MatthewBemis) [inline automation/project/Dependencies.scala:15]: this is my snapshot. it only has Thurloe moved over

Comment (breilly2) [reply automation/project/Dependencies.scala:15]: Is this why "definitely don't merge"?

File: automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/api/sam/SamApiSpec.scala [hunk 2]
```diff
@@ -45,7 +48,7 @@ class SamApiSpec extends FreeSpec with Matchers with ScalaFutures {
     if (Sam.admin.doesUserExist(subjectId).getOrElse(false)) {
       Sam.admin.deleteUser(subjectId)
     }
-    Thurloe.keyValuePairs.deleteAll(subjectId)
+    thurloe.keyValuePairs.deleteAll(subjectId)
```
Comment (breilly2) [inline automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/api/sam/SamApiSpec.scala:51]: Here and elsewhere that uses the new `Thurloe`... need to change the `AuthToken` import to:
```
import org.broadinstitute.dsde.workbench.auth.AuthToken
```

**Review by breilly2 — COMMENTED** [review]
Other than fixing an import, I think this looks good. Do the tests pass?

**Comment by breilly2** [comment]
Actually, I know the tests won't work because they'll need a real `AuthToken` that's currently in Ursa's `2801-sam` PR.

**Pull request closed without merging**